File: fcs59/python/fib.py
n = int(input('Enter a number: '))

a = 0
b = 1

if n < 0:
    print('Enter positive number')
elif n == 0:
    print(0)
else:
    for i in range(n):
        print(a, end=" ")
        next = a + b
        a = b
        b = next

File: fcs59/js/fib.js
let n = 4

let a = 0
let b = 1

if(n < 0){
    console.log('Enter positive number')
}else if(n == 0){
    console.log(0)
}else{
    let result = ''
    for(let i = 0; i < n; i++){
        result += a + ' '
        let c = a + b
        a = b
        b = c
    }
    console.log(result)
}